java-11/build.gradle.kts

```diff
@@ -5,6 +5,6 @@ plugins {
 hypertraceDocker {
   defaultImage {
     imageName.set("java")
-    setTagNameTransform { tag -> "11-${tag.name}" }
+    setTagNameTransform { tag -> "11.0.8" }
```

Comment:
The minor version isn't specified anywhere but in our own tag - so we're assuming that `azul/zulu-openjdk-debian:11` will be on 11.0.8 - which is build time dependent. In other words, this can't be rebuilt consistently. If we think the minor version is important (I'm skeptical), we should parameterize it so we're using a from version that matches our own tag.

Reply:
pinned azul image version in Dockerfile.

Reply:
could we make that a parameter from gradle (it supports build args) - that way, the version is only defined in place and we don't accidentally get out of sync.

Reply:
done.